packages/medusa/src/api-v2/admin/inventory-items/[id]/location-levels/[location_id]/route.ts

```diff
@@ -45,3 +47,37 @@ export const DELETE = async (req: MedusaRequest, res: MedusaResponse) => {
     deleted: true,
   })
 }
+
+export const POST = async (
+  req: MedusaRequest<AdminPostInventoryItemsItemLocationLevelsLevelReq>,
+  res: MedusaResponse
+) => {
+  const { id: inventory_item_id, location_id } = req.params
+  const remoteQuery = req.scope.resolve(ContainerRegistrationKeys.REMOTE_QUERY)
+
+  const { errors } = await updateInventoryLevelsWorkflow(req.scope).run({
+    input: {
+      updates: [{ inventory_item_id, location_id, ...req.validatedBody }],
+    },
+    throwOnError: false,
+  })
+
+  if (Array.isArray(errors) && errors[0]) {
+    throw errors[0].error
+  }
+
+  const [inventory_level] = await remoteQuery(
+    remoteQueryObjectFromString({
+      entryPoint: "inventory_level",
+      variables: {
+        inventory_item_id,
+        location_id,
+      },
+      fields: req.retrieveConfig.select as string[],
```

Comment:
todo: req.remoteQueryConfig.fields, it is important to do the distinction as the value are not quite the same for select all